sdv/constraints/base.py

```diff
@@ -184,6 +184,7 @@ def transform(self, table_data):
         """
         self._use_reject_sampling = False
         self._validate_all_columns_present(table_data)
+        table_data = table_data.copy()
```

Comment:
this may have conflicts when you rebase after the handling strategy issue gets merged. When resolving, can you just move this line to the top of the method?